### PR TITLE
Enable scalastyle for core

### PR DESCRIPTION
**Pull request by rklaehn**
This will prevent style issues for PRs in the future.

This PR also adds a validate command alias, and switches the travis build to the containerized infrastructure, which gives a slight increase in test performance.

### CODE REVIEW

**Comment by non** [comment]
:+1: once this finishes I'll merge it.

**Comment by tixxit** [comment]
:+1:

File: build.sbt [hunk 1]
```diff
@@ -175,6 +175,7 @@ lazy val noPublishSettings = Seq(
   publishLocal := (),
   publishArtifact := false)
 
+addCommandAlias("validate", ";core/scalastyle;compile;test")
```
Comment (tixxit) [inline build.sbt]: I had no idea this existed - awesome!

Comment (rklaehn) [inline build.sbt]: Me neither. I just stole it from the cats build.

**Pull request merged**